data/json/items/melee/spears_and_polearms.json

```diff
@@ -79,6 +79,48 @@
     "qualities": [ [ "CUT", 1 ], [ "BUTCHER", -22 ] ],
     "flags": [ "FRAGILE_MELEE", "NONCONDUCTIVE", "POLEARM", "SHEATH_SPEAR", "REACH_ATTACK", "ALWAYS_TWOHAND" ]
   },
+  {
+    "id": "makeshift_halberd",
+    "type": "GENERIC",
+    "name": { "str": "simple makeshift glaive" },
+    "//": "Name changed to simple makeshift glaive, but changing the id would break e.g.  tilesets.",
+    "description": "This is a large blade attached to a stout section of tree branch.  It could do a considerable amount of damage.",
+    "weight": "1800 g",
+    "volume": "3 L",
+    "longest_side": "180 cm",
+    "price": 5000,
+    "price_postapoc": 250,
+    "to_hit": -1,
+    "bashing": 13,
+    "cutting": 31,
+    "material": [ "steel", "wood" ],
+    "symbol": "/",
+    "color": "light_gray",
+    "techniques": [ "WBLOCK_1" ],
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", -42 ] ],
+    "flags": [ "REACH_ATTACK", "POLEARM", "NONCONDUCTIVE", "SHEATH_SPEAR", "FRAGILE_MELEE" ]
+  },
+  {
+    "id": "makeshift_glaive",
+    "type": "GENERIC",
+    "name": { "str": "makeshift glaive" },
+    "description": "A stout tree branch that has been carefully split and reinforced.  At the split point, a sharp blade has been bolted into place and reinforced with layers of sturdy wrapped bindings.",
+    "weight": "1800 g",
+    "volume": "3 L",
+    "longest_side": "180 cm",
+    "price": 5000,
+    "price_postapoc": 400,
+    "to_hit": -1,
+    "bashing": 13,
+    "cutting": 31,
+    "material": [ "steel", "wood" ],
+    "symbol": "/",
+    "color": "light_gray",
+    "looks_like": "makeshift_halberd",
+    "techniques": [ "WBLOCK_1" ],
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", -42 ] ],
+    "flags": [ "REACH_ATTACK", "POLEARM", "NONCONDUCTIVE", "SHEATH_SPEAR" ]
+  },
```

Comment:
```suggestion
{
    "id": "makeshift_glaive",
    "type": "GENERIC",
	  "copy-from": "makeshift_halberd",
    "name": { "str": "makeshift glaive" },
    "description": "A stout tree branch that has been carefully split and reinforced.  At the split point, a sharp blade has been bolted into place and reinforced with layers of sturdy wrapped bindings.",
    "price_postapoc": 400,
    "delete": { "flags": [ "FRAGILE_MELEE" ] }
  },
```
![image](https://user-images.githubusercontent.com/51099123/113789600-66343a80-970d-11eb-8b65-6ece2fd781d4.png)

Same thing in less lines(ignoring name change).

Reply:
Your changes have been committed, hadn't thought of using copy from, which is indeed much more elegant.